2004-11-08  Richard Sandiford  <rsandifo@redhat.com>

	* sysdeps/mips/sys/regdef.h: Include #include <sgidefs.h>
	* sysdeps/mips/sys/ucontext.h: Likewise.

Index: sysdeps/mips/sys/regdef.h
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/mips/sys/regdef.h,v
retrieving revision 1.4
diff -u -p -F^\([(a-zA-Z0-9_]\|#define\) -r1.4 regdef.h
--- sysdeps/mips/sys/regdef.h	9 Apr 2003 02:51:04 -0000	1.4
+++ sysdeps/mips/sys/regdef.h	28 Oct 2004 12:57:20 -0000
@@ -20,6 +20,8 @@
 #ifndef _SYS_REGDEF_H
 #define _SYS_REGDEF_H
 
+#include <sgidefs.h>
+
 /*
  * Symbolic register names for 32 bit ABI
  */
Index: sysdeps/mips/sys/ucontext.h
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/mips/sys/ucontext.h,v
retrieving revision 1.4
diff -u -p -F^\([(a-zA-Z0-9_]\|#define\) -r1.4 ucontext.h
--- sysdeps/mips/sys/ucontext.h	17 Mar 2003 15:47:12 -0000	1.4
+++ sysdeps/mips/sys/ucontext.h	28 Oct 2004 12:57:20 -0000
@@ -23,6 +23,7 @@ #define _SYS_UCONTEXT_H	1
 
 #include <features.h>
 #include <signal.h>
+#include <sgidefs.h>
 
 /* Type for general register.  */
 #if _MIPS_SIM == _MIPS_SIM_ABI32
